layouts/solo/iframe-static.html

```diff
@@ -78,7 +80,10 @@
 
       return settings;
     }());
-
+    var playlist = settings.playlist || 'hide';
```

Comment:
You can use `settings.playlist` straight or make it null if it doesn;t exists there, like:


```
var playlist = settings.playlist || null;
if (playlist && "show" === playlist) {
  new Menu(player, channelVideos);
}
```